### PR TITLE
WT-9541 Pull out GitHub and Evergreen info from perf_run.py to upload_stats_atlas.py

### PR DESCRIPTION
**Pull request by Siddhartha8899**

### CODE REVIEW

File: test/evergreen.yml
Comment (ajmorton) [inline test/evergreen.yml:735]: Since we're modifying this line can we lock the version of psutil? 
Looking at [one of our recent commits](https://evergreen.mongodb.com/task/wiredtiger_ubuntu2004_perf_tests_perf_test_evict_btree_1_4a23a5846b31a014e2ec5c169bac856c9a942d1f_23_03_08_09_01_09/0) we're currently using version 5.9.4.

**Review by ajmorton — COMMENTED** [review]
LGTM, with one small request.
I'll hold off on approving until we've merged the associated `upload_stats_atlas.py` to be safe

File: test/evergreen.yml
Comment (Siddhartha8899) [reply test/evergreen.yml]: Sure, updated in [835bcbe](https://github.com/wiredtiger/wiredtiger/pull/8904/commits/835bcbe11d2ec56b12b7d16152f07619963578ae)

**Review by ajmorton — APPROVED** [review]
LGTM 👍

**Pull request merged**
